src/sdl/index.ts

```diff
@@ -105,6 +107,28 @@ export class SDL {
     }
   }
 
+  static validateStorage(name: string, storage: v2ResourceStorage | v2ResourceStorageArray | undefined) {
```

Comment:
it might be cool to use schema validation for this

Reply:
We have in the roadmap to improve and add more validation to the SDL parsing. We have to move the validation done in Cloudmos to this repo.